moveit_planners/chomp/chomp_motion_planner/src/chomp_optimizer.cpp

```diff
@@ -429,23 +428,24 @@ bool ChompOptimizer::optimize()
       }
       else
       {
-        // ROS_INFO_STREAM("cCost " << cCost << " over threshold " << parameters_->getCollisionThreshold());
+        // RCLCPP_INFO(LOGGER,"cCost " << cCost << " over threshold " << parameters_->getCollisionThreshold());
       }
     }
 
-    if ((ros::WallTime::now() - start_time).toSec() > parameters_->planning_time_limit_)
-    {
-      ROS_WARN("Breaking out early due to time limit constraints.");
-      break;
-    }
+    // TODO
```

Comment:
See my comment about using `chrono`

Reply:
Looks like I skipped this one during my previous walk through, fixed now.